### ISSUE TITLE
Next Episodes wrong date 

### ISSUE THREAD

**Issue by unicornsandcandies** [Tikipeter/repository.tikipeter#26]
On both of my devices Next Episodes section in Fen acts like it has a wrong date. Show airing today have "tomorrow" air date and do not appear as released until next day. I've checked this with my regional settings and everything is okay. Also everything works in TMDB Helper on Next Episodes from trakt page. It looks like Fen can't grab a proper date from Kodi or is purposefully delaying shows airing by a day

**Comment by Tikipeter** [comment]
Fen neither grabs a date from Kodi nor does it purposely delay dates by 24 hours.

It sounds like you need to correctly set your UTC offset in Fen's settings.

**Comment by Tikipeter** [comment]
And please post a log with your next reply, so as the rules are followed.

**Comment by unicornsandcandies** [comment]
> And please post a log with your next reply, so as the rules are followed.

What log should I post for this? I live in Poland, it's UTC +2 and that's how I've set this up in Fen so that's not the issue

**Comment by Tikipeter** [comment]
A Kodi log.

Then change the UTC value until ithe airdates show correctly.

**Comment by unicornsandcandies** [comment]
> 

I've tried this already
[kodi (copy).log](https://github.com/Tikipeter/repository.tikipeter/files/9481499/kodi.copy.log)




Here's the log:

**Comment by Tikipeter** [comment]
>I've tried this already

You've tried all 31 positions on the UTC range and none of them make the episode show the correct date?

**Comment by unicornsandcandies** [comment]
> > I've tried this already
> 
> You've tried all 31 positions on the UTC range and none of them make the episode show the correct date?

Yes

**Comment by Tikipeter** [comment]
Well, I'm out of ideas then....

**Comment by xyzfre** [comment]
> > > I've tried this already
> > 
> > 
> > You've tried all 31 positions on the UTC range and none of them make the episode show the correct date?
> 
> Yes

After changing the UTC did you try clearing cache afterwards

**Comment by Tikipeter** [comment]
No need to clear the cache, but I did have a thought that if he/she was checking a widget item after changing the setting then it wouldnt reflect the new value until the widgets were reloaded. Inside Fen the values would change immediately.

**Comment by ambulancePilot** [comment]
Follow up question to this. I'm not experiencing the same problem as this guy, but if a show is airing at 9:00 p.m. on Monday, it comes up on the next episode list at 12:00 a.m. on Monday. Is this intended behavior? Should it not wait until 9:00 p.m. on Monday to show the episode in the next episodes list?

**Comment by Tikipeter** [comment]
Unfortunately, TMDb does not provide airtimes for episodes. So they are pretty much irrelevant in Fen.

I could ask Trakt for the info, as they (currently) provide that info, but there are a couple of good reasons that's not going to happen.

Often there are trade offs needed with these addons. I decided right from the beginning to rely on no metadata from Trakt and rely solely on TMDb. TVDb was added later, but then removed when they announced they were going behind a paywall. So now only TMDb is used once again.
